DataFormats/TrackerCommon/interface/TrackerDetSide.h

```diff
@@ -0,0 +1,8 @@
+#ifndef DataFormats_TrackerCommon_TrackerDetSide_H
+#define DataFormats_TrackerCommon_TrackerDetSide_H
+
+struct TrackerDetSide {
+  enum Side { Barrel = 0, NegEndcap = 1,  PosEndcap = 2 };
```

Comment:
Is anything else expected inside ```struct TrackerDetSide```?
this could be ```enum class ```

Reply:
I thought I had a reason for using unscoped `enum`, but I see in the end there is only one place converting it to/from a number and that is already using `static_cast`. Will change to `enum class` (after you complete the review).

Reply:
There were more conversions from the enum to an unsigned int in `HitExtractor*`. Something that could be also considered is to propagate this enum to `TrackerTopology` (as the numbering is intentionally consistent with `TrackerTopology::side()` (but that is beyond this PR).